### PR TITLE
en,zh: release notes for 1.3.8

### PR DESCRIPTION
**Pull request by KanShiori**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/releases/release-1.3.8.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# TiDB Operator 1.3.8 Release Notes
+
+发布日期：2022 年 9 月 13 日
+
+TiDB Operator 版本：1.3.8
+
+## 新功能
+
+- 为 `TidbCluster` 添加一些特殊的 Annotation 以支持配置 TiDB、TiKV 和 TiFlash 的 Pod 的最小等待时间，最小等待时间指的是在滚动升级过程中新创建的 Pod 的变为 Ready 所需要的最小时间 ([#4640](https://github.com/pingcap/tidb-operator/pull/4640), [@KanShiori](https://github.com/KanShiori))
```
Comment (csuzhangxc) [inline zh/releases/release-1.3.8.md:13]: 是不是还得在文档哪里进一下怎么配置这个 😢

Comment (KanShiori) [reply zh/releases/release-1.3.8.md:13]: 是，现在最新功能都没有写文档，最近准备恶补下文档..

Comment (KanShiori) [reply zh/releases/release-1.3.8.md:13]: 先记个 issue https://github.com/pingcap/docs-tidb-operator/issues/1996

Comment (ran-huang) [inline zh/releases/release-1.3.8.md:13]:
```suggestion
- 为 `TidbCluster` 添加一些特殊的 Annotation 以支持配置 TiDB、TiKV 和 TiFlash 的 Pod 的最小等待时间，最小等待时间指的是在滚动升级过程中新创建的 Pod 变为 Ready 所需要的最小时间 ([#4640](https://github.com/pingcap/tidb-operator/pull/4640), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.3.8.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# TiDB Operator 1.3.8 Release Notes
+
+Release date: September 13, 2022
+
+TiDB Operator version: 1.3.8
+
+## New Features
+
+- Add some special annotaions for TidbCluster to configure min ready duration for TiDB, TiKV and TiFlash, it specifies that the minimum number of seconds for which a newly created Pod should be ready during rolling upgrade ([#4675](https://github.com/pingcap/tidb-operator/pull/4675), [@liubog2008](https://github.com/liubog2008))
```
Comment (ran-huang) [inline en/releases/release-1.3.8.md:13]:
```suggestion
- Add some special annotaions for TidbCluster to configure the minimum ready duration for TiDB, TiKV, and TiFlash. The minimum ready duration specifies the minimum number of seconds that a newly created Pod takes to be ready during a rolling upgrade ([#4675](https://github.com/pingcap/tidb-operator/pull/4675), [@liubog2008](https://github.com/liubog2008))
```

File: en/releases/release-1.3.8.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Improvement
+
+- Support graceful upgrade TiCDC pods if pod version is equal or larger than 6.3.0 ([#4697](https://github.com/pingcap/tidb-operator/pull/4697), [@overvenus](https://github.com/overvenus))
```
Comment (ran-huang) [inline en/releases/release-1.3.8.md:17]:
```suggestion
- Support graceful upgrade of a TiCDC Pod if the Pod version is v6.3.0 or later versions ([#4697](https://github.com/pingcap/tidb-operator/pull/4697), [@overvenus](https://github.com/overvenus))
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**